src/main/java/com/autotune/analyzer/services/UpdateRecommendations.java

```diff
@@ -109,21 +109,18 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
                     statusValue = KruizeConstants.APIMessages.SUCCESS;
                 } else {
                     LOGGER.error(String.format(AnalyzerErrorConstants.APIErrors.UpdateRecommendationsAPI.UPDATE_RECOMMENDATIONS_FAILED_COUNT, calCount));
-                    LOGGER.error(String.format(KruizeConstants.APIMessages.UPDATE_RECOMMENDATIONS_FAILURE, experiment_name, intervalEndTimeStr));
-                    sendErrorResponse(response, null, kruizeObject.getValidation_data().getErrorCode(), kruizeObject.getValidation_data().getMessage());
+                    sendErrorResponse(response, null, kruizeObject.getValidation_data().getErrorCode(), kruizeObject.getValidation_data().getMessage(), experiment_name, intervalEndTimeStr);
                 }
             } else {
                 LOGGER.error(String.format(KruizeConstants.APIMessages.UPDATE_RECOMMENDATIONS_VALIDATION_FAILURE, validationMessage,
```

Comment:
Can this LOGGER.error be removed as well?

Reply:
Removed the redundant error log - as the the same `validationMessage` is passed to `sendErrorResponse()`